UnityProject/Assets/Scripts/Health/CPRable.cs

```diff
@@ -62,15 +62,17 @@ private void ServerDoCPR(GameObject performer, GameObject target,BodyPartType Ta
 	{
 		var health = target.GetComponent<LivingHealthMasterBase>();
 
-		health.CirculatorySystem.ConvertUsedBlood(OXYLOSS_HEAL_AMOUNT);
-
 		foreach (var BodyPart in health.GetBodyPartsInZone(TargetBodyPart))
 		{
 			var heart = BodyPart as Heart;
 			if (heart != null)
 			{
 				heart.Heartbeat(HeartbeatStrength);
 			}
+
+			// Still violates thermodynamics, but meh.
+			BodyPart.BloodContainer.CurrentReagentMix.Add(health.CirculatorySystem.CirculatedReagent,
```

Comment:
you could check for lung and then do it to that?

Reply:
There is quite a bit of delay between the lung pulling in oxygen and body parts receiving them, so I was thinking this way would make CPR more effective, but that would be more realistic, yeah.  Also, gave me a good idea, one moment.